ibtk/include/ibtk/ibtk_utilities.h

```diff
@@ -61,55 +61,12 @@ IBTK_ENABLE_EXTRA_WARNINGS
 #if __has_cpp_attribute(deprecated)
 #define IBTK_DEPRECATED(msg) [[deprecated(msg)]]
 #else
-#define IBTK_DEPRECATED(msg)
+#define IBTK_DEPRECATED(msg) [[deprecated]]
 #endif
 #else
 #define IBTK_DEPRECATED(msg)
 #endif
 
-#define IBTK_DEPRECATED_CLASS1(deprecated_class_name)                                                                  \
```

Comment:
One of the rationales for these macros is to standardize output and minimize text in setting up deprecation notices. Could we keep that strategy even if we move to an all-header-based approach?

Reply:
We could, but that makes it harder to read the header itself. I'm ambivalent about this trade-off.